source/develop/release-management/features/storage/incremental-backup.html.md

```diff
@@ -28,138 +25,246 @@ allow incremental backup to start.
 *   Daniel Erez (<derez@redhat.com>)
 
 
-## Incremental backup flow
-Backup flow is the same from the point of view of the user (backup
-application), but will be implemented differently under the hood.
+## High level Design
+
+### Creating VM
+
+There no special configuration for enabling incremental backup, but only
+qcow2 disks support incremental backup.
+
+To enable incremental backup for preallocated disk (raw format), create
+a VM snapshot. This adds a qcow2 image supporting incremental backup.
+Backing up a preallocate disk will preform a full backup.
+
```

Comment:
We should check if bitmaps are always enabled once a checkpoint
is created, or we need to specify that a VM is enabled for incremental
backup. In the last case we will have to add configuration to VM to enable
this feature.